configuration-sample/ods-core.env.sample

```diff
@@ -234,3 +234,5 @@ LOGGING_FILE_PATH=/opt/provision/history/logs
 # If you use or might use documentation templates in the future,
 # it is highly recommended to add "ods-document-generation-templates" to this list.
 READABLE_OPENDEVSTACK_REPOS=ods-jenkins-shared-library,ods-quickstarters
+
+WEBHOOK_PROXY_EVENTS=repo:refs_changed,pr:merged,pr:declined,pr:deleted
```

Comment:
Cool. Could you add some description? E.g.:
```
# Comma-separated list of events for which Bitbucket sends a request to the webhook proxy service
# to create/start a Jenkins pipeline or delete a Jenkins pipeline.
# Pipelines can be created/started for: repo:refs_changed, pr:opened
# Pipelines will be deleted for: pr:merged, pr:declined, pr:deleted
```

Reply:
description makes sense to me. I have included now in the 3rd commit